arcgis-ios-sdk-samples.xcodeproj/project.pbxproj

```diff
@@ -882,6 +886,8 @@
 		12BF9E441FE9D77000B1E6C4 /* ViewshedCameraViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = ViewshedCameraViewController.swift; sourceTree = "<group>"; };
 		166E8F9B1EE62E0E00C57C46 /* RasterLayer.storyboard */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = file.storyboard; path = RasterLayer.storyboard; sourceTree = "<group>"; };
 		166E8F9C1EE62E0E00C57C46 /* RasterLayerViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = RasterLayerViewController.swift; sourceTree = "<group>"; };
+		169CDBC9223C6B5F00ABFF0A /* AddPointSceneLayer.storyboard */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = AddPointSceneLayer.storyboard; path = "../Add a point scene layer/AddPointSceneLayer.storyboard"; sourceTree = "<group>"; };
+		169CDBCB223C6B8400ABFF0A /* AddPointSceneLayerViewController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; name = AddPointSceneLayerViewController.swift; path = "arcgis-ios-sdk-samples/Scenes/Add a point scene layer/AddPointSceneLayerViewController.swift"; sourceTree = SOURCE_ROOT; };
```

Comment:
Please update this file's location to be relative to the group rather than the project.

![Screen Shot 2019-03-19 at 9 04 40 AM](https://user-images.githubusercontent.com/2257493/54622069-19963d80-4a26-11e9-99ea-4535bd7d8a18.png)

When the group's path has been fixed, the path for this (and the storyboard) should just be the file name.